readthedocs/restapi/templates/restapi/log.txt

```diff
@@ -0,0 +1,13 @@
+RTD build information
+Build: {{ build.pk }}
+Project: {{ build.project.slug }}
+Version: {{ build.version.verbose_name }} ({{ build.commit }})
```

Comment:
@humitos I just realized that here we don't have the project slug and version verbose name

Reply:
We only have the project id and commit

Reply:
Why? Take a look at your first screenshot and log attached: there were present.

We should have that information since we pass a `build` object to the template and the build is associated to a `project`, so we should be able to get that information (which is important).

Reply:
Because before this template was called from the Build object (raw_log property), but now, we uses the `data` variable, which contains the data specified on the serializer. We could get that info again by retrieving the object from the id given by the serializer, but I'm not sure if that's ok.

Reply:
I think making a new query is fine. This won't be a heavily hit API or anything, so I'm not too worried about performance.